### DISCUSSION TITLE
Create nested object dynamically

### DISCUSSION THREAD

**Discussion by Helioswei** [nlohmann/json#2675] [Q&A] [answered]
I want to Create object dynamically.For example: the config file with json is 

```json
{
	"name": "helios",
	"persion": {
		"age": 22,
		"height": 180
	}
}
```

I can read the json key and value with iterator。When I want to set value with key persion.age,how can i locate the place。
I know I can set value using json["persion"]["age"],but In the config file ,I don't know How many levels are there? json[][][]...n

**Comment by nlohmann** [comment, marked as answer]
Maybe the `items` function can help: https://json.nlohmann.me/api/basic_json/items/
